files/group

```diff
@@ -70,3 +70,4 @@ tracing:x:341:
 radiusd:x:301:
 postgres:x:28:
 mail:x:8:
+arpwatch:x:402:
```

Comment:
Where did you get these values from?

Reply:
I just reserved an available static ID to arpwatch without which the extra feeds build would fail.

Reply:
Not sure if this is the right thing to do but can merge this for now. We'll have to take a closer look at how best to fix this. Please create a WI for it.

Reply:
It's not unusual that we have to define a uid/gid for a service that is getting pulled into the feeds. Because we chose to use the OE static_uidgid configuration, we have to define all users and groups at build time. The numbers we choose to assign them are arbitrary and only follow the numbering range policy that we defined, and is in comments at the top of the file. FWIW, nothing about his change seems odd to me.